Client/core/DXHook/CProxyDirect3D9.cpp

```diff
@@ -166,6 +168,20 @@ HRESULT CProxyDirect3D9::CreateDevice(UINT Adapter, D3DDEVTYPE DeviceType, HWND
     SetWindowTextW(hFocusWindow, MbUTF8ToUTF16("MTA: San Andreas").c_str());
     #endif
 
+    // Set dark titlebar if needed
+    HKEY hKey;
+    DWORD systemThemeMode = 1;
+    DWORD dataSize = sizeof(systemThemeMode);
+
+    if (RegOpenKeyExW(HKEY_CURRENT_USER, L"Software\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize", 0, KEY_READ, &hKey) == ERROR_SUCCESS)
```

Comment:
There is the dedicated MTA function for this: `ReadRegistryStringValue`

Reply:
Thanks, done